conf.yaml

```diff
@@ -2,11 +2,30 @@ plugins:
   trivy:
     exit-code: 0
     timeout: 5m0s
-    severity: HIGH
+    severity: HIGH,CRITICAL
     ignore-unfixed: true
     security-checks: vuln,config
     skip-files: ""
     skip-dirs: ""
     image-ref: ""
     version: ""
-    helm-overrides-file: ""
\ No newline at end of file
+    helm-overrides-files: ""
+  cosign:
+    image: "ghcr.io/my-project/my-image:latest"
+    keyless: true
```

Comment:
we would want to be careful about this not being the default - ie, for private/internal repos we'd use private/internal sigstore instance

Reply:
Addressed, I updated and added keyless=false as a default.